### PR TITLE
Basetemplate

### PR DESCRIPTION
**Pull request by dbursem**
I'm working on a new template. It's work in progress but i'd like to hear your thoughts on it. I used http://getskeleton.com/ css framework (that's a first for me). 

The reason I started this work is because I believe the DDB website should be a lot more informative about the reason and function of the DDB. 

it can be seen live on http://ddb.dbursem.duckdns.org:9998

### CODE REVIEW

**Comment by snip** [comment]
Looks very good! I love it.
Thanks a lot for the works done.
Waiting forward for the completion.

**Comment by wbuczak** [comment]
Hi David

I also like it very much!

Just minor remark - in the description:

*This is the place to register your glider, towplane or other
FLARM-equipped aircraft to the Open Glider *

 you mention FLARM-equipped devices, but DDB can/is also hosting OGN
trackers.

Cheers
Wojtek

**Comment by dbursem** [comment]
I think the template is pretty much finished as is. 

Only things needed are French + German translations for some new content. I've put the English version in the language files for somebody to translate. Also maybe a native English speaker should have a look at my English version. 

Should I merge upstream in this branch or is it better to let the upstream maintainer do all the merging?

**Comment by snip** [comment]
Yes please merge with latest upstream and resolve conflicts.
Thanks

**Comment by kerel-fs** [comment]
Hi @dbursem,
I recommend you to rebase your branch (and force-push it) like I did with your last PullRequest.
This way you're also able to squash the commits into one commit per feature
(e.g. `index.php: Use $REQUEST instead of $POST` and `Add base template with skeleton.css`).

A guide about rebasing by Atlassian: [merging vs rebasing](https://www.atlassian.com/git/tutorials/merging-vs-rebasing/)

File: templates/base.html.twig [hunk 1]
```diff
@@ -1,32 +1,57 @@
...
-{% block content %}{% endblock %}
-
-
-<br><br><center>
-{%  block disconnect %}{% endblock %}
-    <img src="/pict/ogn-logo-ani.gif"></center>
+<div class="container">
+    <div class="row header">
+        <div class="ten columns">
+            <img class="logo" src="pict/ogn-logo.png"><h1>  Devices DataBase</h1>
```
Comment (kerel-fs) [inline templates/base.html.twig]: @dbursem `pict/ogn-logo.png` doesn't exist

File: index.php [hunk 2]
```diff
@@ -257,7 +256,7 @@ function devicelist() {
 		fromhome();
 		if (!isset($_SESSION['login'])) exit(); // test if user come from login page
 		$_SESSION['dev'] = 'yes';
-		if (isset($_POST['devid'])) $devid=$_POST['devid'];
+		if (isset($_REQUEST['devid'])) $devid=$_REQUEST['devid'];
```
Comment (kerel-fs) [inline index.php]: @dbursem Why should we change this?

**Comment by kerel-fs** [comment]
Nice new templates! Especially the description for  "Full participation", "Anonymous Participation" and "No Participation" are an improvement.
I tested it a bit in a local vagrant box, beside the comments above, looks good for me.

Maybe someone from the mailing list will improve the translations. :wink:

**Comment by dbursem** [comment]
Ok I finally merged upstream and added the ODB-BY license link. Think it's ready for production now, although German and French language files are not complete yet.

**Comment by snip** [comment]
Thanks dbursem.
Before putting it in production, i discovered following issues:
- it is not possible to update a registered device
- it is not possible to delete a registered device

Can you please correct this?

Thanks

**Comment by dbursem** [comment]
Thank you, i'm gonna test it.

Met vriendelijke groet,

David Ursem
Op 18 jun. 2016 11:34 schreef "Sebastien Chaumontet" <
notifications@github.com>:

**Comment by dbursem** [comment]
Hi Snip,

Fixed this and another issue with the last commits. Tested for a bit and couldn't find any other issues.

**Comment by snip** [comment]
Merged and moved to production.
Thanks a lot David for this new UI!

**Pull request merged**